integration-tests/spring-cloud-config-client/src/test/java/io/quarkus/spring/cloud/config/client/runtime/SpringCloudConfigServerResource.java

```diff
@@ -0,0 +1,79 @@
+package io.quarkus.spring.cloud.config.client.runtime;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+
+import org.apache.commons.io.IOUtils;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
```

Comment:
Are you sure about this?

I don't ever remembering us using this server. I'd much rather use Wiremock

Reply:
I can certainly revert to Wiremock.

I prefer to use this because it doesn't require additional dependencies, setup on the build, and proprietary configuration files.

Reply:
Sure yeah, but it's theoretically an internal class, right?